lib/fizzy/execute.cpp

```diff
@@ -356,6 +362,62 @@ T fnearest(T value) noexcept
         return t;
 }
 
+
+template <typename T>
+T fabs(T) noexcept = delete;
```

Comment:
@gumb0, @axic I'm currently in opinion that `abs()` is better name as we don't have `iadd()` and `fadd()`, just `add()`.

Reply:
I was actually going to suggest to keep the `f` prefix and also use `fneg`, given these functions are specifically working on floating points. Since there's no `abs` on integers in Wasm, it doesn't come up as an issue. Should there be one in the future, happy to rename it to `abs`.

Reply:
Same for `fcopysign` then?

Reply:
https://en.cppreference.com/w/cpp/numeric/math/copysign seems to be floating point only

Reply:
Keeping `fcopysign` then.